compiler/natives/src/internal/reflectlite/reflectlite.go

```diff
@@ -0,0 +1,973 @@
+// +build js
+
+package reflectlite
+
+import (
+	"unsafe"
+
+	"github.com/gopherjs/gopherjs/js"
+)
+
+var initialized = false
+
+func init() {
+	// avoid dead code elimination
+	used := func(i interface{}) {}
+	used(rtype{})
+	used(uncommonType{})
+	used(method{})
+	used(arrayType{})
+	used(chanType{})
+	used(funcType{})
+	used(interfaceType{})
+	used(mapType{})
+	used(ptrType{})
+	used(sliceType{})
+	used(structType{})
+	used(imethod{})
+	used(structField{})
+
+	initialized = true
+	uint8Type = TypeOf(uint8(0)).(*rtype) // set for real
```

Comment:
Is there magic happening that I'm not seeing, that requirese this to happen in `init()`?

Reply:
Same as above, this code was cargo-culted over from `reflect` and it seems to work, so I opted against touching it for now. Once GopherJS is fully up to date with Go 1.16 and is stable we could could try and experiment with removing potentially load-bearing artifacts :)